Dockerfile

```diff
@@ -1,19 +1,13 @@
-FROM node:10
-# Create Remix user, don't use root!
-# RUN yes | adduser --disabled-password remix && mkdir /app
-# USER remix
-
-# #Now do remix stuff
-# USER remix
-WORKDIR /home/remix
-
-RUN git clone https://github.com/ethereum/remix-ide.git
-RUN git checkout origin remix_live
-
-WORKDIR /home/remix/remix
-RUN npm install
-RUN npm run build
-
-EXPOSE 8080 65520
-
-CMD ["npm", "run", "serve"]
+FROM nginx:alpine
+WORKDIR /
+
+COPY ./build/ /usr/share/nginx/html/build/
+COPY ./index.html /usr/share/nginx/html/index.html
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./assets/ /usr/share/nginx/html/assets/
+COPY ./icon.png /usr/share/nginx/html/icon.png
+COPY ./background.js /usr/share/nginx/html/background.js
+COPY ./soljson.js /usr/share/nginx/html/soljson.js
+COPY ./package.json /usr/share/nginx/html/package.json
+
+EXPOSE 80
```

Comment:
@yann300 this is a variant of dockerfile which just copies all necessary files from host to the docker container without building anything.